pygmt/src/text.py

```diff
@@ -33,7 +35,7 @@
     e="find",
     f="coltypes",
     h="header",
-    i="incols",
+    i="use_word",
```

Comment:
We are aliasing `-it` to `word`, so need to change `i` to `it` here:
```suggestion
    it="use_word",
```

Reply:
I try to work on this over the weekend. I am sorry for the delay.